modules/virt-reclaiming-persistent-volumes-for-binding-to-a-new-persistent-volume.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// virt/virtual_machines/virtual_disks/virt-reclaiming-persistent-volumes.adoc
+
+[id="virt-reclaiming-persistent-volumes-for-binding-to-a-new-persistent-volume_{context}"]
+
+= Reclaiming persistent volumes for binding to a new persistent volume
+
+Use this procedure to reclaim a statically-provisioned persistent volume for reuse.
+
+.Procedure
+. Enter the following command to display connection details for the persistent volume
+you want to reclaim. Save the output for future reference.
++
+----
+$ oc describe pv <pv_name>
+----
+. Edit the `storageclass.yaml` file. Verify that the Storage Class `reclaimPolicy` is set to `delete`.
+. Enter the following command to verify that no other resources
+are using the persistent volume claim:
++
+----
+$ oc describe pvc <pvc_name>
+----
+. Delete the persistent volume claim to release the persistent volume:
++
+----
+$ oc delete pvc/<volume_claim>
+----
+. Delete the persistent volume:
++
+----
+$ oc delete pv/<volume>
+----
+. Delete all data in the network share:
++
+----
+$ rm -rf <path_to_share>
+----
+. Create a persistent volume that uses a different volume name. When creating the
+new volume, use the same connection details as the deleted persistent volume.
+.. Enter the following command to edit the PV's configuration:
++
+----
+$ oc get pv <name> -o yaml > pv.yaml
+----
+.. Edit `pv.yaml`
+.. Change the name of PV in the YAML file to a different name.
+.. Delete the PV you are no longer using:
++
+----
+$ oc delete pv/<volume>
+----
```

Comment:
Are you actually creating a PV? Don't you have to use the modified YAML to create a new object?

Reply:
Reworded. Hopefully this is OK. I want to be sure they know that the new name is of most importance.

Reply:
My problem with this section is that it says that you're going to create a PV, but I don't see where you do it. You're exporting the PV definition, but I don't see where you're reusing it.